jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java

```diff
@@ -0,0 +1,169 @@
+        return new Object[][]{
+                {dropdownForm.form.email, "email", "Email address", "email@example.com"},
+                {dropdownForm.form.password, "password", "Password", "Password"}
+        };
+    }
+
+    @Test(dataProvider = "listFields")
+    public void isValidationTests(TextField element, String type, String label, String value) {
+        dropdownForm.expand();
+
+        element.clear();
+        element
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("form-control")
+                .attr("type", type)
+                .attr("placeholder", value)
+                .tag(is("input"));
+        element.label()
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .text(is(label));
+
+        dropdownForm.collapse();
+    }
+
+    @Test
+    public void isValidationTests() {
+        dropdownForm.expand();
+
+        dropdownForm.form.radio
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("form-check")
+                .tag(is("div"));
+        dropdownForm.form.radio.label()
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("form-check-label")
+                .text(is("Remember me"))
+                .tag(is("label"));
+
+        dropdownForm.collapse();
+    }
+
+    @Test
+    public void checkboxTests() {
+        dropdownForm.expand();
+
+        dropdownForm.form.radio.check();
+        dropdownForm.form.radio.is().selected();
+        dropdownForm.form.radio.uncheck();
+        dropdownForm.form.radio.is().deselected();
+
+        dropdownForm.collapse();
+    }
+
+    @Test
+    public void fillTest() {
+        dropdownForm.expand();
+
+        dropdownForm.form.fill(USER);
+        dropdownForm.form.check(USER);
+
+        dropdownForm.collapse();
+    }
+
+    @Test
+    public void testButton(){
+        dropdownForm.expand();
+
+        dropdownForm.form.submit();
+        validateAlert(is("Sign in"));
+
+        dropdownForm.collapse();
+    }
+
+    @DataProvider(name = "linksDataProvider")
+    public Object[][] linksDataProvider() {
+        return new Object[][]{
+                {"New around here? Sign up", "https://jdi-testing.github.io/jdi-light/index.html"},
+                {"Forgot password?", "https://jdi-testing.github.io/jdi-light/index.html"},
+
+        };
+    }
+
+    @Test(dataProvider = "linksDataProvider")
+    public void linkItemsTest(String itemText, String itemHref) {
+        dropdownForm.expand();
+
+        int currWindowNum = WindowsManager.windowsCount();
+        dropdownForm.select(itemText);
+
+        WindowsManager.switchToWindow(currWindowNum + 1);
+        String url = WebPage.getUrl();
+        Assert.assertEquals(url, itemHref);
```

Comment:
assertEquals(url, itemHref) (without Assert , in other places we use this style)